Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -3082,7 +3083,7 @@ namespace AssetProcessor
                 jobIter = m_jobsToProcess.erase(jobIter);
                 m_numOfJobsToAnalyze--;
```

Comment:
is this just white space changes only?

Reply:
yeah, I added a newline before the for loop, it was a bit easier for me to read like that.